canvas_modules/common-canvas/__tests__/test_resources/paramDefs/structurelisteditor_paramDef.json

```diff
@@ -10,6 +10,34 @@
         "World",
         "string",
         "Readonly phrase"
+      ],
```

Comment:
@tanvimehta10 Can you make these same changes to the corresponding structurelisteritor_paramDef in the test harness? We want the test files to be consistent to what we see in the test harness

Reply:
Done